docs/data/sdks/ios-swift/ampli.md

```diff
@@ -0,0 +1,206 @@
+---
+title: iOS Ampli Wrapper
+description: Learn how to install and use the Ampli Wrapper for the iOS Swift runtime. 
+---
+
+--8<-- "includes/ampli/ampli-overview-section.md"
+
+Amplitude Data supports tracking analytics events from iOS apps written in Swift.
+
+!!!warning "Objective-C Not Supported"
+    Please note that the [latest iOS SDK](/data/sdks/ios-swift) is NOT compatible with Objective-C projects. Use the [maintenance iOS SDK](/data/sdks/ios) with [maintenance Ampli](/data/sdks/ios/ampli) if your project requires compatibility with Objective-C.
+
+!!!info "Ampli iOS Resources"
+    [:material-language-swift: Swift Example](https://github.com/amplitude/ampli-examples/tree/main/ios/swift/v2/AmpliSwiftSampleApp) · [:material-code-tags-check: Releases](https://www.npmjs.com/package/@amplitude/ampli?activeTab=versions)
+
+--8<-- "includes/ampli-vs-amplitude-link-to-core-sdk.md"
+    Visit the [Amplitude iOS SDK](./index.md) documentation.
+
+## Quick Start
+
+0. [(Prerequisite) Create a Tracking Plan in Amplitude Data](https://help.amplitude.com/hc/en-us/articles/5078731378203)
+
+    Plan your events and properties in [Amplitude Data](https://data.amplitude.com/). See detailed instructions [here](https://help.amplitude.com/hc/en-us/articles/5078731378203)
+
+1. [Install the Amplitude SDK](#install-the-amplitude-sdk)
+
+    ```shell
+    pod 'AmplitudeSwift', '~> 0.4.14'
+    ```
+
+2. [Install the Ampli CLI](#install-the-ampli-cli)
+
+    ```shell
+    npm install -g @amplitude/ampli
+    ```
+
+3. [Pull the Ampli Wrapper into your project](#pull)
+
+    ```shell
+    ampli pull [--path ./Ampli]
+    ```
+
+4. [Initialize the Ampli Wrapper](#load)
+
+    ```swift
+    Ampli.instance.load(LoadOptions(
+      environment: AmpliEnvironment.Production
+    ))
+    ```
+
+5. [Identify users and set user properties](#identify)
+
+    ```swift
+    Ampli.instance.identify("userID", Identify(
+        userProp: "A trait associated with this user"
+    ))
+    ```
+
+6. [Track events with strongly typed methods and classes](#track)
+
+    ```swift
+    Ampli.instance.songPlayed(SongPlayed(songId: "song-1")
+    Ampli.instance.track(SongFavorited(songId: "song-2")
+    ```
+
+7. [Flush events before application exit](#flush)
+
+    ```swift
+    Ampli.instance.flush()
+    ```
+
+8. [Verify implementation status with CLI](#status)
+
+    ```shell
+    ampli status [--update]
+    ```
+
+## Installation
+
+### Install the Amplitude SDK
+
+If you haven't already, install the core Amplitude SDK dependencies.
+
+@{% from 'sdks/ios-install-dependencies.md' import ios_install_dependencies %}
+@{{ios_install_dependencies()}}
+
+--8<-- "includes/ampli/cli-install-simple.md"
+
+## API
+
+### Load
+
+Initialize Ampli in your code. The `load()` method accepts configuration option arguments:
```

Comment:
Can we mention they can access to every method of the underlying Amplitude SDK here like [here](https://www.docs.developers.amplitude.com/data/sdks/browser-2/ampli/#api)?

Reply:
Done.